chat-plugins/wisp.js

```diff
@@ -1095,6 +1095,50 @@ exports.commands = {
 		"/background set [user], [image] - Sets a users profile background.",
 		"/background delete [user] - Deletes a users profile background.",
 	],
+	music: {
+		set: function (target, room, user) {
+			if (!this.can('music')) return false;
+			if (!target) return this.parse('/help music');
+			let targets = target.split(',');
+			for (let u in targets) targets[u] = targets[u].trim();
+			if (!targets[1]) return this.parse('/help music');
+
+			let targetUser = Users(targets[0]);
+			let song = targets[1];
+
+			if (!targetUser || !targetUser.connected) return this.errorReply(target[0] + " is not online");
+			if (!song) return this.errorReply("Please specify a song to set.");
+			if (song.length > 100) return this.errorReply("Music URLs may not be longer than 100 characters.");
+
+			Wisp.setMusic(targetUser.userid, song);
+			Wisp.messageSeniorStaff("/html " + Wisp.nameColor(user.name, true) + " has set " + Wisp.nameColor(targetUser.name, true) + "'s profile music to: " + Tools.escapeHTML(song));
+			Rooms('upperstaff').add("|raw|" + Wisp.nameColor(user.name, true) + " has set " + Wisp.nameColor(targetUser.name, true) + "'s profile music to: " + Tools.escapeHTML(song));
+
+			this.sendReply("You've set " + targetUser.name + "'s profile music.");
+		},
+
+		delete: function (target, room, user) {
+			if (!this.can('music')) return false;
+			if (!target) return this.parse('/help music');
+
+			let targetUser = Users(target);
+
+			if (!targetUser || !targetUser.connected) return this.errorReply(target[0] + " is not online");
+
+			Wisp.setMusic(targetUser.userid, "");
+			Wisp.messageSeniorStaff("/html " + Wisp.nameColor(user.name, true) + " has removed " + Wisp.nameColor(targetUser.name, true) + "'s profile music");
+			Rooms('upperstaff').add("|raw|" + Wisp.nameColor(user.name, true) + " has removed " + Wisp.nameColor(targetUser.name, true) + "'s profile music");
+
+			this.sendReply("You've removed " + targetUser.name + "'s profile music.");
+		},
+		'': function (target, room, user) {
+			return this.parse('/help music');
+		},
+	},
+	musichelp: [
+		"/music set [user], [song] - Sets a users profile music.",
```

Comment:
and do you want the format changed on all help objects?  Because most use the format I submitted